From a2a59e18e53510ce7c3010a47fb3d633e72c737b Mon Sep 17 00:00:00 2001
From: Frank Wunderlich <frank-w@public-files.de>
Date: Mon, 17 May 2021 18:54:55 +0200
Subject: [PATCH 1036/1041] net: dsa: mt7530: fix compile error

dsa_switch does not contain assisted_learning_on_cpu_port yet,
so disable line added by multi cpu patch
---
 drivers/net/dsa/mt7530.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/dsa/mt7530.c b/drivers/net/dsa/mt7530.c
index 0b38099bb02c..e71b04615679 100644
--- a/drivers/net/dsa/mt7530.c
+++ b/drivers/net/dsa/mt7530.c
@@ -2756,7 +2756,7 @@ mt7530_probe(struct mdio_device *mdiodev)
 	priv->dev = &mdiodev->dev;
 	priv->ds->priv = priv;
 	priv->ds->ops = &mt7530_switch_ops;
-	priv->ds->assisted_learning_on_cpu_port = true;
+	//priv->ds->assisted_learning_on_cpu_port = true;
 	mutex_init(&priv->reg_mutex);
 	dev_set_drvdata(&mdiodev->dev, priv);
 
-- 
2.31.1

